go/vt/vtgate/planbuilder/symboltable.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package planbuilder
+
+import (
+	"errors"
+
+	"github.com/youtube/vitess/go/vt/sqlparser"
+)
+
+// SymbolTable contains the symbols for a SELECT
```

Comment:
I think it's ok to be non-commital and still put that it in the comments. We can change the comments later on when you're more sure.